### PR TITLE
PPP - Johnson Yee

### PR DESCRIPTION
**Pull request by Johnson-Yee**
Regarding #192

### CODE REVIEW

**Review by EthanWong2212 — APPROVED** [review]

File: docs/team/johnsonYee.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+* **Community**:
+  * PRs reviewed (with non-trivial review comments):
+   [\#23](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/23),
+   [\#29](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/29),
+   [\#78](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/78), 
+   [\#195](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/195),
+   [\#205](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/205)
+  * Reported bugs and suggestions for other teams ([Click here](https://github.com/Johnson-Yee/ped/issues)
+   * Reviewed developer guide for team DomNUS ([click here](https://github.com/nus-cs2113-AY2021S1/tp/pull/8/files/65a23531bf8d85984e3d339cfc455cedb7e0cccc)
```
Comment (yuxinwu99) [inline docs/team/johnsonYee.md]: Might be good to standardise using (Pull Request: ), currently some states that it is a PR while some are just # numbers.

Comment (Johnson-Yee) [reply docs/team/johnsonYee.md]: Alright!

File: docs/team/johnsonYee.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+* **Documentation**:
+  * User Guide:
+    * Added documentation for the features `/routemap`, `/reset`, `/deletefav` (Pull requests
+    [#81](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/81),
+    [#87](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/87),
+    [#147](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/147))
+    * Designed Nav@NUS logo used in user guide and developer guide.
+    ([#142](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/142))
+    * Added sections on introduction and quick start. ([#142](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/142))
```
Comment (Johnson-Yee) [inline docs/team/johnsonYee.md]:
```suggestion
    * Added sections on introduction and quick start. (Pull request[#142](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/142))
```

File: docs/team/johnsonYee.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  
+  * Developer Guide:
+    * Added implementation details, class diagrams and sequence diagrams for the `/routemap`, `/deletefav` and `/reset`
+     features. ([#116](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/116)) 
+     !!!!!Remember to update after you finish!!!!
+     * Updated target user profile and value proposition. ([#91](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/91))
```
Comment (Johnson-Yee) [inline docs/team/johnsonYee.md]:
```suggestion
     * Updated target user profile and value proposition. (Pull request[#91](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/91))
```

File: docs/team/johnsonYee.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    [#81](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/81),
+    [#87](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/87),
+    [#147](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/147))
+    * Designed Nav@NUS logo used in user guide and developer guide.
+    ([#142](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/142))
+    * Added sections on introduction and quick start. ([#142](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/142))
+  
+  * Developer Guide:
+    * Added implementation details, class diagrams and sequence diagrams for the `/routemap`, `/deletefav` and `/reset`
+     features. ([#116](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/116))
```
Comment (Johnson-Yee) [inline docs/team/johnsonYee.md]:
```suggestion
     features. (Pull request [#116](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/116)) 
```

File: docs/team/johnsonYee.md (hunk 1 above)
Comment (Johnson-Yee) [inline docs/team/johnsonYee.md]:
```suggestion
   * Reviewed developer guide for team DomNUS ([Click here](https://github.com/nus-cs2113-AY2021S1/tp/pull/8/files/65a23531bf8d85984e3d339cfc455cedb7e0cccc)
```

File: docs/team/johnsonYee.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  * Developer Guide:
+    * Added implementation details, class diagrams and sequence diagrams for the `/routemap`, `/deletefav` and `/reset`
+     features. ([#116](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/116)) 
+     !!!!!Remember to update after you finish!!!!
+     * Updated target user profile and value proposition. ([#91](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/91))
+     
+
+* **Community**:
+  * PRs reviewed (with non-trivial review comments):
+   [\#23](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/23),
```
Comment (Johnson-Yee) [inline docs/team/johnsonYee.md]:
```suggestion
   Pull requests[\#23](https://github.com/AY2021S1-CS2113T-F14-3/tp/pull/23),
```

**Pull request merged**